tools/pd-ctl/pdctl/command/keyspace_group_command.go

```diff
@@ -357,7 +357,7 @@ func showKeyspaceGroupPrimaryCommandFunc(cmd *cobra.Command, args []string) {
 		cmd.Printf("Failed to parse the keyspace group ID: %s\n", err)
 		return
 	}
-	r, err := doRequest(cmd, fmt.Sprintf("%s/%s/primary", keyspaceGroupsPrefix, args[0]), http.MethodGet, http.Header{})
+	r, err := doRequest(cmd, fmt.Sprintf("%s/%s?get_param=primary", keyspaceGroupsPrefix, args[0]), http.MethodGet, http.Header{})
```

Comment:
I think it should be a bool, right?

Reply:
"%s/%s?only_primary_member=" so?

Reply:
Maybe you can google it to find the best practice?

Reply:
maybe it is better, "%s/%s?fields=primary" and return `{ id:0, primary:""}`
ref: 
https://stackoverflow.com/questions/18235546/way-to-specify-resources-fields-list-in-restful-api-request
https://stackoverflow.com/questions/68979696/restful-api-including-additional-fields